pkg/api/validate/dynamic/quota.go

```diff
@@ -88,6 +88,19 @@ func addRequiredResources(requiredResources map[string]int, vmSize api.VMSize, c
 	case api.VMSizeStandardGS5:
 		requiredResources["standardGSFamily"] += (count * 32)
 		requiredResources["cores"] += (count * 32)
+		// Support for L series VMs
+	case api.VMSizeStandardL4sV2:
```

Comment:
Refer to comment [here](https://github.com/Azure/ARO-RP/pull/1751/files#r713123628)